docs/api/instruments/teledyne/teledyneT3AFG.rst

```diff
@@ -5,4 +5,4 @@ Teledyne T3AFG Arbitrary Waveform Generator
 .. autoclass:: pymeasure.instruments.teledyne.teledyneT3AFG
     :members:
     :show-inheritance:
-    :inherited-members:
+    :inherited-members: Instrument
```

Comment:
Why is that still Instrument?
Did I miss anything?

Reply:
I think it would be easier if https://github.com/pymeasure/pymeasure/pull/941 is merged first because it fixes a few files, like this one. After that I can go through and update `:inherited-members:` to be `:inherited-members: CommonBase` throughout the .rst files.

Reply:
Ok, let's do 941 first.

Reply:
Updated that doc